packages/wallets/src/features/cfd/screens/MT5TradeScreen/MT5TradeScreen.scss

```diff
@@ -21,11 +31,18 @@
             justify-content: space-between;
             width: 100%;
 
-            &-title {
+            #{$root}__label {
                 display: flex;
                 flex-direction: column;
             }
 
+            #{$root}__title {
```

Comment:
To avoid deeply nesting the classnames, I reused the grandparent classname by assigning it to the `#{$root}` variable to name my class

Reply:
typically its recommended to keep nesting in BEM to single level, 

e.g. instead of: 
.SomeComponent__block1__block2

easier to have:
.SomeComponent__block1
.SomeComponent__block2

(no need for extra variables and easier to read and reuse, as block2 now can be used outside of block1 context)

Reply:
oh yeahh it doesn't need to be nested at all 😅